pkg/ecdsa/tss/tss_celo_test.go

```diff
@@ -0,0 +1,11 @@
+//+build celo
+
+package tss
+
+import (
+	"github.com/celo-org/celo-blockchain/crypto"
+	"github.com/celo-org/celo-blockchain/crypto/secp256k1"
+)
+
+var S256 = secp256k1.S256
```

Comment:
Could use a quick comment here to indicate why this file exists.

Reply:
Please see https://github.com/keep-network/keep-ecdsa/commit/382ba37582d155d0735476a65518b831f17dca52